index.js

```diff
@@ -26,6 +25,20 @@ bot.on("error", err => {
     console.log(err);
 });
 
+bot.on("guildMemberAdd",(guild, member) =>{
+    let userName = member.username
+    let cocChannel = process.env.COC_CHANNEL_ID
+    let serverName = guild.name
+    bot.getDMChannel(member.id).then(
```

Comment:
I'd make this an async method and use await here, since I doubt Eris uses the return value of this callback for anything.